pennylane/ops/qubit/parametric_ops.py

```diff
@@ -728,7 +728,9 @@ def compute_eigvals(theta, n_wires):  # pylint: disable=,arguments-differ
         return qml.math.exp(-1j * theta / 2 * eigs)
 
     @staticmethod
-    def compute_decomposition(theta, wires, **kwargs):  # pylint:disable=unused-argument
+    def compute_decomposition(
+        theta, wires, **kwargs
```

Comment:
Because I want to "hide" kwargs we don't use here, otherwise people wonder why `n_wires` is passed but never needed...I think this is a good pattern going forward!